include/lsst/jointcal/AstrometryTransform.h

```diff
@@ -400,6 +400,17 @@ class AstrometryTransformPolynomial : public AstrometryTransform {
      * @see ast::PolyMap for details of the structure of this matrix.
      */
     ndarray::Array<double, 2, 2> toAstPolyMapCoefficients() const;
+
+    template <class M, class V, class T>
+    double _computeFit(M A, V B, T monomials,
```

Comment:
Please use more descriptive names than `M, V, T` for these template parameters, and for the `A` and `B` variables.